lib/ansible/modules/system/systemd.py

```diff
@@ -48,12 +48,13 @@
         type: bool
         default: 'no'
         aliases: [ daemon-reload ]
-    user:
+    scope:
```

Comment:
I'm fine with adding scope, but user should be deprecated, not removed and the interaction between both handled, with scope having precedence when set.

Reply:
I didn't see in the documentation, but is there a standard way of makring something as deprecated?

Reply:
just add a note about deprecation in the description and call` module.deprecate()` if it is set, our normal deprecation cycle is 4 versions, so if this goes in now (2.7) it should be removed in (2.11 or equivalent)